app/assets/json/translations/locale-es.json

```diff
@@ -1043,6 +1097,14 @@
                 "step_5": "Tenga ahorros suficientes para los costos de cierre y el pago inicial",
                 "title": "Antes de presentar la solicitud"
             },
+            "before_applying_habitat": {
+                "read_full_list": "Para obtener más detalles, lea <a href='{{url}}' target='_blank'>la lista completa de requisitos</a>.",
+                "rules_are_different": "Estas unidades poseen normas diferentes de otros anuncios de propiedad en DAHLIA. Asegúrese de:",
+                "step_1": "asistir a una <a href='{{url}}' target='_blank'>sesión informativa de Habitat for Humanity</a>",
+                "step_2": "no haber sido propietario de una propiedad residencial en los últimos 3 años",
+                "step_3": "cumplir con los <a href='{{url}}' target='_blank'>requisitos de ingresos</a>",
+                "step_4": "tener una puntuación de crédito de 650 o mayor"
```

Comment:
Should the first words for each of these steps be capitalized?

Reply:
just pushed this change